pkg/kubelet/volumes.go

```diff
@@ -285,10 +285,6 @@ func (kl *Kubelet) newVolumeAttacherFromPlugins(spec *volume.Spec, pod *api.Pod,
 	if err != nil {
 		return nil, fmt.Errorf("can't use volume plugins for %s: %v", spec.Name(), err)
 	}
-	if plugin == nil {
```

Comment:
like @pmorie's comment on Detacher below, this is also valid.  Not every volume plugin will have an attacher.